app/styles/molecules/button-dropdown.less

```diff
@@ -0,0 +1,62 @@
+.oss-button-dropdown {
+  position: relative;
+
+  &__trigger {
+    border: 1px solid @upf-primary-rock-blue;
+    border-radius: var(--border-radius-sm);
+    height: 37px;
+    padding: var(--spacing-px-6) 0;
+
+    div {
+      height: 37px;
+      padding: 0 var(--spacing-px-18);
+
+      &:last-child {
+        padding: 0 var(--spacing-px-12);
+        border-left: 1px solid @upf-primary-rock-blue;
```

Comment:
Same here for the color pallet

Reply:
it doesn't much the existing buttons so it looks weird when placed next to existing buttons (checked w/ Victor)